projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/repository/ProjectDataRepository.java

```diff
@@ -0,0 +1,33 @@
+package life.qbic.projectmanagement.domain.project.repository;
+
+import life.qbic.projectmanagement.domain.project.Project;
+import life.qbic.projectmanagement.domain.project.ProjectCode;
+
+
+/**
+ * <b>Project Data Storage Interface</b>
+ *
+ * <p>Provides access to the persistence layer that handles the {@link Project} data storage.
+ *
+ * @since 1.0.0
+ */
+public interface ProjectDataRepository {
+
+  /**
+   * Saves a {@link Project} entity permanently.
```

Comment:
Does it save a project? It rather creates space to store data for a project of a specific code, right?